infoschema/tables.go

```diff
@@ -1353,6 +1357,16 @@ var tableTiDBTrxCols = []columnInfo{
 	{name: "DB", tp: mysql.TypeVarchar, size: 64, comment: "The schema this transaction works on"},
 }
 
+var tableDeadLockCols = []columnInfo{
+	{name: "DEADLOCK_ID", tp: mysql.TypeLonglong, size: 21, flag: mysql.NotNullFlag},
+	{name: "OCCUR_TIME", tp: mysql.TypeTimestamp, decimal: 6, size: 26},
+	{name: "TRY_LOCK_TRX_ID", tp: mysql.TypeLonglong, size: 21, flag: mysql.NotNullFlag},
+	{name: "CURRENT_SQL_DIGEST", tp: mysql.TypeVarchar, size: 64},
+	{name: "KEY", tp: mysql.TypeBlob, size: types.UnspecifiedLength},
+	{name: "SQLS", tp: mysql.TypeBlob, size: types.UnspecifiedLength},
```

Comment:
What's this?

Reply:
They are history SQLs of the transaction?

Reply:
Yes and it's not yet implemented. By the way I'm proposing renaming for it at https://github.com/pingcap/tidb/pull/24718 .